docs/courseadministration/addcontent/learningunits/quiz_import.md

```diff
@@ -0,0 +1,68 @@
+![HPI Logo](../../../img/HPI_Logo.png)
+
+# Quiz Import from Google Spreadsheets
+
+You can import the quiz structure from Google Spreadsheets.
+All you have to do is the following:
+
+## 1. Make a copy of this template
+We can provide you with a quiz template that you can copy
+
+![How our quiz template looks like](../../../img/courseadministration/quizmanagement/quiz_template.png)
+
+### Find the non-to-be-edited quiz structure template [here](https://docs.google.com/spreadsheets/d/1_b8wO9_HlrrHQxagYan1iM6BBK0XtnD6vESxmhnML5s)
+
+### Before proceeding, remember:
+## Please, do not edit directly the template, but make a copy of it - see below how to do it.
+
+![How to make a copy of a Google spreadsheet template](../../../img/courseadministration/quizmanagement/copy_quiz_template.png)
+
+Now that you have created a copy, you can edit yours with the quiz questions for all the weeks and homeworks in your course.
+Please note that every row after the _Quizname_ rows is a question.
+
+### Cheat sheet of quiz questions and answers
+
+##### The infos about the quiz and its type:
+
+- *Name*: the name of the quiz
+- *CourseSection*: the section of the course where the quiz is. It must be a numeric value.
+- *Quiz Type*: choose the most appropriate type for the quiz
+- *Instructions*: The instructions for taking the quiz. Please note that self-tests do not have instructions
+
+##### Question type, its points and explanation (green cells)
+
+- *Question*: actual text of the question
+- *QuestionType*: choose between multiple answer or multiple choice
+- *Points*: points of the question
+- *Explanation*: optional explanation of the question
+
+##### The answers to the question (blue and violet cells)
+
+- *Answer1*, *Answer2*,* Answer3*, *Answer4* : the text of the answer
+- *Correctness* :choose 'True' for a correct answer or  ('False') for a wrong answer
+- *Explanation*: text of the optional explanation of the answer. Similar to the explanation of a question, this is shown together with the results of the quiz.
+
+
+### 2. Share your quiz spreadsheet to our quiz service
+
+:bulb: _Before proceeding with this step, you may want to rename your quiz spreadsheet in a unique way, e.g._: `coursecode_quizname`
```

Comment:
I don't know whether mkdocs support the icon. We need to check that...

Reply:
Obviously, we need [an extension that enables emojis](https://facelessuser.github.io/pymdown-extensions/extensions/emoji/). As long as this isn't included, these won't render. You can try it locally with `mkdocs serve` or just adopt it here.

Reply:
removed ;)

Reply:
Thanks. I am sad to see that go but we can re-add it later ;)